polynote-kernel/src/main/scala/polynote/kernel/logging/package.scala

```diff
@@ -21,8 +21,10 @@ package object logging {
     trait Service {
       def error(msg: String)(implicit location: Location): UIO[Unit]
       def error(msg: Option[String], err: Throwable)(implicit location: Location): UIO[Unit]
+      def errorSync(msg: Option[String], err: Throwable)(implicit location: Location): Unit
```

Comment:
Added these for cases when you need to log from outside an effect (e.g. inside `ref.update`)